react-frontend/src/components/Guides/CollaborationTools/collaborationTools.js

```diff
@@ -1,17 +1,55 @@
 import React from 'react';
-
+import styled from 'styled-components';
+import { createGlobalStyle } from 'styled-components';
 import CollaborationTool from './collaborationTool';
-// import ResourceHeader from './resourceheader';
-// import '@bcgov/bc-sans/css/BCSans.css'
-import '../../../css/collaborationTools.css';
+
+const GlobalStyle = createGlobalStyle`
+ html {
+  font-size: 10px;
```

Comment:
is this the designed base font size?

Reply:
yes this was based on the word press site.  The design guidance was to look as close as possible. This seems to be how wp deals with design